citadel/indico_citadel/search.py

```diff
@@ -64,6 +64,8 @@ def search(self, query, user=None, page=1, object_types=(), **params):
         }
         # Filter by the objects that can be viewed by users/groups in the `access` argument
         if access := get_user_access(user):
+            if allow_admin:
+                access.append('IndicoAdmin')
```

Comment:
We could just use `access = ['IndicoAdmin']` and skip getting the user-specific access altogether

Reply:
`IndicoAdmin` would be above any others privileges the user can have, right?

Reply:
yes, it's set on all records, so passing it makes passing the user-specific data unnecessary